pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/SegmentIndexCreationDriverImpl.java

```diff
@@ -455,6 +459,36 @@ private void handlePostCreation()
     LOGGER.info("Driver, indexing time : {}", totalIndexTime);
   }
 
+  private void updateSegmentStartEndTimeIfNecessary(ColumnStatistics timeColumnStats) {
+    switch (config.getTimeColumnType()) {
+      case EPOCH:
+        break;
```

Comment:
Epoch start/end time is set by default. We are overwriting things if the time format is simpledate